src/pages/iv-ava-by-site/forms/SerialForm.js

```diff
@@ -0,0 +1,74 @@
+import { useContext, useEffect, useState } from 'react'
+import Table from 'src/components/Shared/Table'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useResourceQuery } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+
+const SerialForm = ({ labels, maxAccess, itemId, siteId }) => {
+  const { getRequest } = useContext(RequestsContext)
+  const [data, setData] = useState([])
+
+  async function fetchGridData() {
+    const response = await getRequest({
+      extension: InventoryRepository.AvailabilitySerial.qry,
+      parameters: `_itemId=${itemId}&_siteId=${siteId}&_srlNo=&_startAt=0&_pageSize=50`
+    })
+    setData(response)
+  }
+
+  useEffect(() => {
```

Comment:
why u are using both useEffect and useResourceQuery
remove useEffect it's useless here